docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -179,19 +192,37 @@ The AI Assistant uses functions to include relevant context in the chat conversa
 
 The following table lists available functions:
 
+//TODO: Resolve this question:
+**REVIEWERS**: Is the list of functions here correct? Only a subset are actually shown in the UI. None of the apm functions are shown in the UI right now.
```

Comment:
Yes, some functions are hidden from the user as they're only available to the Assistant, and APM functions only show up when there is APM data in the cluster.